skyrim-platform/src/platform_se/codegen/convert-files/Definitions.txt

```diff
@@ -473,6 +473,39 @@ export const enum DxScanCode {
     MouseWheelDown
 }
 
+export const enum ActorValue {
+    Health = 'health',
+    Magicka = 'health',
+    Stamina = 'health',
```

Comment:
health health health? 💊 🧑‍⚕️ 💉 👨‍⚕️

Reply:
btw we better name av's as they are originally named. **and originally they start with uppercase**

Reply:
we also better add all AVs available https://en.uesp.net/wiki/Skyrim_Mod:Actor_Value_Indices
and copy comments from UESP/CK wiki